cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```
@@ -1055,7 +1055,8 @@ data:
             add_header 'Access-Control-Allow-Origin' '*' always;
             add_header 'Access-Control-Allow-Methods' 'GET, PUT, POST, DELETE, OPTIONS' always;
             return 200 '\n
-                {"ssoConfigured": "{{ template "ssoEnabled" . }}"}\n
+                {"ssoConfigured": "{{ template "ssoEnabled" . }}"},\n
+                {"dataBackupConfigured": "{{ template "dataBackupConfigured" . }}"}\n
```

```suggestion
                {
                "ssoConfigured": "{{ template "ssoEnabled" . }}",
                "dataBackupConfigured": "{{ template "dataBackupConfigured" . }}"
                }
```

I think (?) we should be returning a single JSON object and not two?